priv/js/app.js

```diff
@@ -58,14 +53,15 @@ function reset(siteName) {
 
 async function teleportToSite() {
   let name = siteSearch.value.toLowerCase()
-  if (await isValidSite(name)) {
-    reset(name)
+  const validatedSiteName = await validateSite(name)
+  if (validatedSiteName === "error") {
+    document.querySelector('#helper').hidden = false
+  } else {
+    reset(validatedSiteName)
```

Comment:
And this guy makes it so that our JS frontend only ever starts with a valid sitename, which prevents that initial issue where our first node has some `id` that's actually some alternate version of a valid site name